modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/common.clj

```diff
@@ -22,6 +22,10 @@
   {:pre [(map? db-details) (seq service-account-json)]}
   (ServiceAccountCredentials/fromStream (ByteArrayInputStream. (.getBytes service-account-json))))
 
+(defn database-details->credential-project-id [details]
```

Comment:
public function should probably have a docstring? I'm surprised CI didn't catch this, I guess the linters don't run on driver code?

Reply:
Oops, good catch.  Updating docstrings in next push.